src/util/image/MapCreator.java

```diff
@@ -544,7 +554,20 @@ private static String getValue(final String arg) {
 
   private static void handleCommandLineArgs(final String[] args) {
     final String[] properties = getProperties();
-    if (args.length == 1) {
+    boolean laf = false;
```

Comment:
Does appear to be used actually, but it's not very clean. One if statement sets the value, then another uses it - a bit hard to follow.